tests/split_ncvars/split_ncvars

```diff
@@ -26,6 +26,10 @@ fi
 
 . ${srcdir=.}/init.sh; path_prepend_ ../src
 
+# Set paths to list-vars script and just-compiled exec
+export LIST_NCVARS=`find ../.. -name list_ncvars.sh`
```

Comment:
These should be added to the Makefile.am where the test environment is defined, not here.

Reply:
This is true for all other files.  

PR #338 has a fix for the `PKGLIBEXECDIR`.  Please work to have that merged as well.